pkg/controllers/cstorvolumeconfig/volume_operations.go

```diff
@@ -361,7 +361,12 @@ func (c *CVCController) distributeCVRs(
 	}
 
 	if len(usablePoolList.Items) < pendingReplicaCount {
-		return errors.New("not enough pools available to create replicas")
+		return errors.Errorf(
+			"not enough pools available are available of CSPC: %s usable pool count %d to create %d replicas",
```

Comment:
available repeated ?
change to  --> "not enough pools are available of provided CSPC: " ?

Reply:
How about this?
```go
"not enough pools available are available of CSPC: %s usable pool count %d to create %d replicas"
```

Reply:
> available repeated ? change to --> "not enough pools are available of provided CSPC: " ?

I'm getting this exact error, the only place ive found on good with these words is here, i cant find a fix! 
Any thoughts ?